app-text/alanvar/alanvar-0.5.0.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Al-Anvar, a Quran Study Software "
+DESCRIPTION="Al-Anvar is a cross-platform Quran Study Software. It uses \
+\"Tanzil.net\" and \"Zekr.org\" database as its database."
+
+HOMEPAGE="http://al-anvar.sourceforge.net/"
+COPYRIGHT="2012-2018 S. Hosein Hoseini"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/mazbrili/al-anvar/archive/$portVersion.zip"
+CHECKSUM_SHA256="3f3e57d453302fe11b715e75bce0b2c630cae70143987cf4208c5d8285a52817"
+SOURCE_DIR="al-anvar-$portVersion"
+ADDITIONAL_FILES="alanvar.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	alanvar$secondaryArchSuffix = $portVersion
+	app:Al_anvar = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+  	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+  	lib:libphonon4qt5$secondaryArchSuffix
+  	lib:libQtSql$secondaryArchSuffix
+  	lib:libsqlite3$secondaryArchSuffix
+  	lib:libQt5Xml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+ 	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+  	devel:libQtSql$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+  	devel:libQt5Xml$secondaryArchSuffix
+  	devel:libz$secondaryArchSuffix
+	devel:libphonon4qt5$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:unzip
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	qmake
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp -f ../bin/al-anvar $appsDir/Al-anvar
+
+	local APP_SIGNATURE="application/x-vnd.al-anvar"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/alanvar.rdef.in > alanvar.rdef
+
+	addResourcesToBinaries alanvar.rdef $appsDir/Al-anvar
```

Comment:
Suggestion:
```diff
-		$portDir/additional-files/alanvar.rdef.in > al-anvar.rdef
+		"$portDir"/additional-files/al-anvar.rdef.in > al-anvar.rdef
 
-	addResourcesToBinaries alanvar.rdef $appsDir/Al-anvar 
+	addResourcesToBinaries al-anvar.rdef "$appsDir"/Al-anvar
- 	addAppDeskbarSymlink $appsDir/Al-anvar Al-anvar
+ 	addAppDeskbarSymlink "$appsDir"/Al-anvar Al-anvar
```
(The quotes I've added here and there are not required right now but they might be some day.)